cmd/state-offline-installer/runInstall.go

```diff
@@ -0,0 +1,187 @@
+	err = unzip(params.backpackZipFile, assetsPath)
+	if err != nil {
+		return errs.Wrap(err, "Error unzipping assets")
+	}
+	out.Print(fmt.Sprintf("Stage 1 of 2 Finished: Decompressing assets into: %s", assetsPath))
+
+	archivePath := filepath.Join(assetsPath, artifactsTarGZName)
+
+	out.Print(fmt.Sprintf("Stage 2 of 2 Start: Decompressing artifacts into: %s", artifactsPath))
+	err = unpackArtifact(unarchiver.NewTarGz(), archivePath, artifactsPath)
+	if err != nil {
+		return errs.Wrap(err, "Could not unpack artifact %s", archivePath)
+	}
+	out.Print(fmt.Sprintf("Stage 2 of 2 Finished: Decompressing artifacts into: %s", artifactsPath))
+
+	offlineTarget := target.NewOfflineTarget(installToDir, artifactsPath)
+
+	// FIX: Need to do this more correctly with a New function
+	mockProgress := &OfflineProgressOutput{out: out}
+	// mockProgress := offlineProgressOutput.New(out)
+	logfile, err := buildlogfile.New(outputhelper.NewCatcher())
+	if err != nil {
+		return errs.Wrap(err, "Unable to create new logfile object")
+	}
+	eventHandler := events.NewRuntimeEventHandler(mockProgress, nil, logfile)
+
+	rti, err := runtime.New(offlineTarget, analytics, nil)
+	if err != nil && runtime.IsNeedsUpdateError(err) {
+		err = rti.Update(nil, eventHandler)
+		if err != nil {
+			return errs.Wrap(err, "Had an installation error")
+		}
+	}
+
+	configureEnvironmentAccepted, err := prompter.Confirm("Setup", "Setup environment for installed project?", &default_boolean_answer)
+	if err != nil {
+		return errs.Wrap(err, "Error getting confirmation")
+	}
+
+	if configureEnvironmentAccepted {
+		// This is the code to setup the environment when we install...
+		cfg, err := config.New()
+		if err != nil {
+			return errs.Wrap(err, "Error configuring environment")
+		}
+
+		env, err := rti.Env(false, false)
+		if err != nil {
+			return errs.Wrap(err, "Error setting environment")
+		}
+
+		sshell := subshell.New(cfg)
+
+		if rt.GOOS == "windows" {
+			contents, err := assets.ReadFileBytes("scripts/setenv.bat")
+			if err != nil {
+				return errs.Wrap(err, "Error reading file bytes")
+			}
+			err = fileutils.WriteFile(filepath.Join(installToDir, "setenv.bat"), contents)
+			if err != nil {
+				return locale.WrapError(err, "err_deploy_write_setenv", "Could not create setenv batch scriptfile at path: {{.V0}}", installToDir)
+			}
+		}
+
+		err = sshell.WriteUserEnv(cfg, env, sscommon.DeployID, true)
+		if err != nil {
+			return locale.WrapError(err, "err_deploy_subshell_write", "Could not write environment information to your shell configuration.")
+		}
+
+		binPath := filepath.Join(offlineTarget.Dir(), "bin")
+		if err := fileutils.MkdirUnlessExists(binPath); err != nil {
+			return locale.WrapError(err, "err_deploy_binpath", "Could not create bin directory.")
+		}
+
+		namespace_to_use := project.NewNamespace("owner", "project", "commitID")
+
+		// Write global env file
+		err = sshell.SetupShellRcFile(binPath, env, *namespace_to_use)
```

Comment:
This will lead to `[owner/project]` being part of their prompt. I think you'll want to update `SetupShellRCFile()` to respect a nil value for the namespace and skip that prompt logic.